physical/raft/raft.go

```diff
@@ -919,6 +937,12 @@ func (b *RaftBackend) Get(ctx context.Context, path string) (*physical.Entry, er
 // Put inserts an entry in the log for the put operation
 func (b *RaftBackend) Put(ctx context.Context, entry *physical.Entry) error {
 	defer metrics.MeasureSince([]string{"raft-storage", "put"}, time.Now())
+
+	valueLen := len(entry.Value)
+	if uint64(valueLen) > b.kvMaxValueSize {
+		return fmt.Errorf("value size too large; got %d bytes, max: %d bytes", valueLen, b.kvMaxValueSize)
```

Comment:
We have a constant for these types of errors here:

https://github.com/hashicorp/vault/blob/954c6d98d588fa0776d7d0db8a4eefef19f8fb97/sdk/physical/physical.go#L23

And you can see it being used here:

https://github.com/hashicorp/vault/blob/954c6d98d588fa0776d7d0db8a4eefef19f8fb97/physical/consul/consul.go#L230

I would still embed the valueLen and max length in the error message but if we wrap it with the constant then we can more easily recover from max size errors up the stack.

Reply:
You may also want to push this logic down into the `applyLog` function to cover Transaction calls as well.

Reply:
ACK - Thanks for pointing this out 👍

Reply:
> You may also want to push this logic down into the `applyLog` function to cover Transaction calls as well.

@briankassouf do you mean that we should loop over the command ops and check each value before calling `(Chunking)Apply`?